Menus/TitleScreen.h

```diff
@@ -0,0 +1,289 @@
+		float m_StationOrbitProgress; //!< Progress the station made on the orbit, from 0.0 to 1.0;
+		float m_OrbitRadius; //!< The radius of the planet orbit.
+		MOSParticle m_GameLogo; //!< The title screen scene Cortex Command logo.
+		MOSParticle m_GameLogoGlow; //!< The title screen scene Cortex Command logo glow.
+		float m_GameLogoMainMenuOffsetY; //!< The game logo position in the main menu.
+		float m_GameLogoPlanetViewOffsetY; //!< The game logo position in planet overview menus.
+
+		TitleTransition m_TitleTransitionState; //!< The current title transition (scroll) state.
+
+		bool m_SectionSwitch; //!< Whether a new section or a transition has started and new properties need to be applied.
+		Timer m_SectionTimer; //!< Timer to keep track of the section elapsed time.
+		float m_SectionElapsedTime; //!< How many seconds have elapsed on a section.
+		float m_SectionDuration; //!< How many seconds a section is supposed to elapse.
+		float m_SectionProgress; //!< Progress made on a section, from 0.0 to 1.0.
+
+		Vector m_ScrollOffset; //!< The scrolling position of the whole title screen scene.
+		float m_BackdropScrollRatio; //!< The scrolling ratio of the whole title screen scene background (nebula and stars).
+		float m_PlanetViewScrollOffsetY; //!< The title screen scene scroll position when in planet overview menus.
+
+		bool m_FinishedPlayingIntro; //!< Whether the intro sequence finished playing.
+		IntroSequence m_IntroSequenceState; //!< The current intro sequence state.
+		Timer m_IntroSongTimer; //!< Timer used to sync the intro sequence progress with the intro song.
+		float m_IntroScrollStartTime; //!< Time point during the intro sequence where the title screen scene scrolling starts.
+		float m_IntroScrollDuration; //!< How many seconds the duration of the title screen scene scrolling should elapse during the intro sequence.
+		float m_IntroScrollStartOffsetY; //!< The title screen scene starting scroll position during the intro sequence.
+		float m_GameLogoAppearScrollOffsetY; //!< The title screen scene scroll position at which the game logo appears during the intro sequence.
+		float m_PreMainMenuScrollOffsetY; //!< The title screen scene scroll position before scrolling to and activating the main menu.
+		float m_SlideFadeInDuration; //!< How many seconds the duration of a slideshow slide fade in is supposed to elapse.
+		float m_SlideFadeOutDuration; //!< How many seconds the duration of a slideshow slide fade out is supposed to elapse.
+		std::unique_ptr<GUIFont> m_IntroTextFont; //!< The GUIFont used for drawing text during the logo splash screens and slideshow.
+		std::string m_SlideshowSlideText; //!< String containing the slide text during each section of the slideshow.
+		BITMAP *m_DataRealmsLogo; //!< The DataRealms logo bitmap used in the logo splash screen.
```

Comment:
No.
I think it's kinda pointless and annoying to specify since they almost always not.
It would be better to specify when they are.
E.g created/loaded with `create_bitmap`/`load_bitmap` or loaded through `ContentFile` without passing ownership to the static maps (which these do exactly that here so I screwed up a lil bit but it's fixed now).

Reply:
Okay, so these are owned by contentfile static maps? Makes sense since, as you say, that's the norm.
I think it is worth specifying though, since you can choose to not have your bitmap be owned by the static maps, though I'm not sure how often we do that.

But yeah, even very low level stuff like TerrainObject specifies that it's not owned here. I know it's a pain, but ownership stuff is important to keep clear, and this helps avoid a little bit of potential confusion.